### DISCUSSION TITLE
How to add a share button to every blog post

### DISCUSSION THREAD

**Discussion by olaoluwasalami** [facebook/docusaurus#9565] [General] [answered]
Is there a way to add a share button to every blog post using docusaurus.config.js?

**Comment by slorber** [comment]
You can do that by swizzling the blog components, not through the site config

**Reply by olaoluwasalami** [reply]
Thanks @slorber, In previous versions of Docusaurus,  there's an optional field `facebookAppId` and `twitter` that shows a share button at the end of each blog post, how is that achievable with Docusarurus.config.js without swizzling or do I create a site.config.js for this?

**Reply by slorber** [reply, marked as answer]
We don't support that v1 anymore. 
Swizzle is the way to achieve it and is much more flexible than what we had before even though it's more work for you to set it up
